day3.html

```diff
@@ -123,7 +123,8 @@
     <ul class="steps">
       <li>Dépendance = nom du package + contrainte de version</li>
       <li><code>x.y.z</code> : exactement la version</li>
-      <li><code>^x.y.z</code> : la version x la plus haute (y.z au minimum)</li>
+      <li><code>^x.y.z</code> : le premier chiffre différent de 0 est fixé (les autres chiffres au minimum)</li>
```

Comment:
https://nodesource.com/blog/semver-tilde-and-caret
>For caret ranges, only major version must match. Any minor or patch version greater than or equal to the minimum is valid.

Cette formulation est plus claire non (je trouve minimum, maximum pas trop top comme image) ? Pareil pour le `tilde`  en dessous.